metrics/metrics.go

```diff
@@ -240,8 +243,9 @@ func (m *Metrics) MeasureAllFiltersResponse(routeId string, start time.Time) {
 }
 
 func (m *Metrics) MeasureResponse(code int, method string, routeId string, start time.Time) {
+	method = measuredMethod(method)
+	m.measureSince(fmt.Sprintf(KeyResponseCombined, code, method), start)
```

Comment:
these actuator metrics also cause some explosion of metrics. I'd also add a flag for it. This is more useful with APIs than regular websites, hence the flag